db/knex_init_db.js

```diff
@@ -86,7 +86,7 @@ async function createTables() {
         table.text("accepted_statuscodes_json").notNullable().defaultTo("[\"200-299\"]");
         table.string("dns_resolve_type", 5);
         table.string("dns_resolve_server", 255);
-        table.string("dns_last_result", 255);
+        table.string("dns_last_result", 2000);
```

Comment:
you are changing an already existing migration. Please add a new migration instead.

I also don't think that this solves the problem of too large results getting stored. => please truncate what we try to store here instead

Reply:
> Please add a new migration instead.

This was the first thing I did on my side. Added a file called `2024-07-11-0000-dns-results.js` with the following content:

```javascript
exports.up = function (knex) {
  return knex.schema.table('monitor', function (table) {
    table.string('dns_last_result', 2000).alter();
  });
};

exports.down = function (knex) {
  return knex.schema.table('monitor', function (table) {
    table.string('dns_last_result', 255).alter();
  });
};

```

But this code did not get executed: https://github.com/louislam/uptime-kuma/blob/master/server/database.js#L393-L396, it seems it just happens when the DB is initialized, not every time the server start (or by a trigger): https://github.com/louislam/uptime-kuma/blob/1a5a1a6e5d019b3dc58d114327f6765da0bee5f5/server/server.js#L1671

Reply:
FYI @CommanderStorm, the migration issue I'm facing is using `mariadb` connection type